handbook/technical-overview/auth/configuration.md

```diff
@@ -200,3 +200,14 @@ Even though the `name` field of clients, resources and scopes all have the same
      "amr": ["external"]
    }
    ```
+
+### Example sign-out
+
+1. User is using the site `https://app.myorg.is` which has the `@myorg.is/webapp` client.
+
+2. The user clicks a logout button. The client redirects them to the IAS endsession endpoint: `https://innskra.island.is/connect/endsession` with these query parameters:
+
```

Comment:
The spec defines more parameters the client could use:
https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout.
The `state` can be used by the client to pass some state information into the post_logout_redirect_uri page, so I think we should at least add the state parameter to the list as an optional parameter.